tests/test_meetup.py

```diff
@@ -25,6 +25,7 @@
 import httpretty
 import os
 import pkg_resources
+import requests
```

Comment:
`requests` is not part of the standar library. It should be imported in other block below.